frontend/app/src/components/inputs/AssetSelect.vue

```diff
@@ -113,12 +115,16 @@ export default defineComponent({
     });
 
     const getAvailableAssets = () => {
+      const unIgnoredAssets = get(supportedAssets).filter(
+        asset => get(showIgnored) || !get(isAssetIgnored(asset.identifier))
+      );
+
       if (get(items) && get(items).length > 0) {
```

Comment:
No need to do anything in this PR but I have a small request for the future. When using `get` to unref a property in the context of a function or a computed property if it is used multiple times instead of doing `get(x)` multiple items can we extract it to a const instead and use the const